runtime/runtime/src/congestion_control.rs

```diff
@@ -36,7 +36,11 @@ pub(crate) struct ReceiptSinkV1<'a> {
 /// receiving shard and stopping us from sending more receipts to it than its
 /// nodes can keep in memory.
 pub(crate) struct ReceiptSinkV2<'a> {
-    pub(crate) congestion_control: &'a mut CongestionControl,
+    /// Keeps track of the local shard's congestion info while adding and
+    /// removing buffered or delayed receipts. At the end of applying receipts,
+    /// it will be a field in the [`ApplyResult`]. For this chunk, it is not
+    /// used to make forwarding decisions.
+    pub(crate) own_congestion_info: &'a mut CongestionInfo,
```

Comment:
It seems like it's the 4th time that we're changing this in the past two weeks :) Hopefully we're getting close :)